Gemfile

```diff
@@ -2,6 +2,11 @@ source "https://rubygems.org"
 gemspec :name => "jekyll"
 
 gem "rake", "~> 11.0"
+
+if RUBY_ENGINE == 'ruby' && RUBY_VERSION < '2.2.2'
+  gem "activesupport", "< 5", :groups => [:test_legacy, :site]
```

Comment:
What from `:site` depends on this gem?

I'd love to see this only take up one line.

Probably best to use `~> 4.2` instead of `< 5`?